src/aws/osml/tile_server/viewpoint/queue.py

```diff
@@ -25,9 +25,9 @@ def __init__(self, aws_sqs, queue_name: str) -> None:
 
         self.sqs_client = aws_sqs
         self.queue = self.sqs_client.get_queue_by_name(QueueName=queue_name)
-        self.logger = logging.getLogger("uvicorn")
+        self.logger = logger
 
-    def send_request(self, request: Dict) -> None:
+    def send_request(self, request: Dict, attributes: Dict = None) -> None:
         """
         Send the message to an associated SQS queue.
 
```

Comment:
Update the docstring to reflect on the new parameters.

Reply:
fixed